user-service/src/test/java/rs/edu/raf/banka/user_service/UserControllerTest.java

```diff
@@ -641,15 +641,15 @@ public static String asJsonString(final Object obj) {
 
     String initValidJWT(){
         return JWT.create()
-                .withSubject(dummyName)
+                .withSubject(dummyName+",ADMIN_ROLE")
```

Comment:
Zašto ne stavimo rolu kao zaseban claim? Mislim da je less error-prone, a osim toga, možda i lakše za parsiranje/upotrebu (npr. za frontend).